### PR TITLE
feat(whitelist): add whitelist to tieraccess middleware

### PR DESCRIPTION
**Pull request by qingyashu**
Add whitelist to tieraccess middleware that will skip encrypting some values. See README for detail. 

### Improvements
  - add whitelist to tieraccess middleware that will skip encrypting some values

### CODE REVIEW

File: src/server/config.js [hunk 1]
```diff
@@ -70,6 +71,13 @@ if (process.env.TIER_ACCESS_LEVEL) {
   config.tierAccessLevel = process.env.TIER_ACCESS_LEVEL;
 }
 
+// check whitelist is disabled
+if (config.encryptWhitelist === 'disabled') {
```
Comment (mfshao) [inline src/server/config.js]: I'm a bit on the fence about if it is okay to use `disabled` as a keyword to disable the whitelist, because what if user wants to use `disabled` as an actual keyword in the whitelist?

I guess it is okay because you can put it like `"encrypt_whitelist": [ "disabled" ]`, but you can also do sth like `"encrypt_whitelist": "bla"`.

Personally I would prefer to have another boolean `enable_encrypt_whitelist` field in config, so that we don't have any special words here. But I'm just being picky here 😈

File: src/server/logger.js [hunk 2]
```diff
@@ -34,6 +34,7 @@ log.levelEnums = {
   SILENT: 5,
 };
 
+log.setLevel('INFO');
```
Comment (mfshao) [inline src/server/logger.js]: why is this line necessary?

File: src/server/config.js (hunk 1 above)
Comment (qingyashu) [reply src/server/config.js]: 😂 cool ok.. haha

File: src/server/logger.js (hunk 2 above)
Comment (qingyashu) [reply src/server/logger.js]: seems the leveled log has default level and won't output my first INFO log in config.js... so I just hard set the level here
